ui/app/routes/vault/cluster/access/oidc/clients/client/providers.js

```diff
@@ -3,6 +3,7 @@ import Route from '@ember/routing/route';
 export default class OidcClientProvidersRoute extends Route {
   model() {
     const model = this.modelFor('vault.cluster.access.oidc.clients.client');
+    // TODO CMB: broken - waiting for backend to accept GET + ?list=true
```

Comment:
Worth noting that "broken" here just means not filtering as expected, not breaking the UI completely 😄

Reply:
haha! good point.. 😅